Automattic/wp-calypso
# FSE: Rename Posts Carousel to Post Carousel

#### Changes proposed in this Pull Request

Rename "Posts Carousel" block to "Post Carousel". Discussion in pbAPfg-no-p2#comment-1012



#### Testing instructions



* Verify name change in the block picker.
* Verify name and description change in the Post Carousel block's sidebar.

See https://github.com/Automattic/newspack-blocks/pull/485
